spec.emu

```diff
@@ -351,23 +351,125 @@ contributors: Ron Buckton, Ecma International
         [+Await] AwaitExpression[?Yield]
 
       <ins class="block">
-        ThrowExpressionInvalidPunctuator : one of `,` `&lt;` `>` `&lt;=` `>=` `==` `!=` `===` `!==` `+` `-` `*` `/` `%` `**` `&lt;&lt;` `>>` `>>>` `&` `|` `^` `&&` `||` `??` `=` `+=` `-=` `*=` `/=` `%=` `**=` `&lt;&lt;=` `>>=` `>>>=` `&=` `|=` `^=` `&&=` `||=` `??=` `?`
+        ThrowExpressionInvalidPunctuator : one of `,` `&lt;` `>` `&lt;=` `>=` `==` `!=` `===` `!==` `*` `%` `**` `&lt;&lt;` `>>` `>>>` `&` `|` `^` `&&` `||` `??` `=` `+=` `-=` `*=` `%=` `**=` `&lt;&lt;=` `>>=` `>>>=` `&=` `|=` `^=` `&&=` `||=` `??=` `?`
       </ins>
     </emu-grammar>
+
+    <emu-clause id="sec-throw-operator">
+      <h1><ins>The `throw` Operator</ins></h1>
+      <emu-clause id="sec-throw-operator-runtime-semantics-evaluation">
+        <h1>Runtime Semantics: Evaluation</h1>
+        <emu-grammar>UnaryExpression : `throw` UnaryExpression</emu-grammar>
+        <emu-alg>
+          1. Let _exprRef_ be ? Evaluation of |UnaryExpression|.
+          1. Let _exprValue_ be ? GetValue(_exprRef_).
+          1. Return ThrowCompletion(_exprValue_).
+        </emu-alg>
+      </emu-clause>
+    </emu-clause>
   </emu-clause>
 
-  <emu-clause id="sec-throw-operator">
-    <h1><ins>The `throw` Operator</ins></h1>
-    <emu-clause id="sec-throw-operator-runtime-semantics-evaluation">
-      <h1>Runtime Semantics: Evaluation</h1>
-      <emu-grammar>UnaryExpression : `throw` UnaryExpression</emu-grammar>
-      <emu-alg>
-        1. Let _exprRef_ be ? Evaluation of |UnaryExpression|.
-        1. Let _exprValue_ be ? GetValue(_exprRef_).
-        1. Return ThrowCompletion(_exprValue_).
-      </emu-alg>
+
+  <emu-clause id="sec-multiplicative-operators">
+    <h1>Multiplicative Operators</h1>
+    <emu-clause id="sec-assignment-operators-static-semantics">
+      <h1>Static Semantics</h1>
+      <emu-clause id="sec-assignment-operators-static-semantics-early-errors">
+        <h1>Static Semantics: Early Errors</h1>
+        <emu-grammar>
+          MultiplicativeExpression :
+            MultiplicativeExpression MultiplicativeOperator ExponentiationExpression
+        </emu-grammar>
+        <ul>
+          <li>
+            It is a Syntax Error if the derived |MultiplicativeExpression| is <emu-grammar>UnaryExpression : `throw` UnaryExpression</emu-grammar> and the source text matched by |MultiplicativeOperator| is `/`.

```

Comment:
```js
a ** throw b * c
```

We must not only check MultiplicativeExpression itself, but whatever ends up being its right-most leaf.

Reply:
`throw b * c` would be banned by ThrowExpressionInvalidPunctuator

Reply:
Ah, I see what you mean.

Reply:
I added a `ContainsThrowOnRight` SDO, though it probably covers more than is strictly necessary.